Cargo.toml

```diff
@@ -116,6 +117,9 @@ reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth", tag = "v1
 reth-trie-common = { git = "https://github.com/paradigmxyz/reth", tag = "v1.9.3" }
 reth-trie-db = { git = "https://github.com/paradigmxyz/reth", tag = "v1.9.3" }
 
+# alethia reth
+alethia-reth-node-builder = { path = "crates/node-builder" }
+
```

Comment:
Do we really need this in `alethia-reth`'s `Cargo.toml`?

Reply:
Use relative paths instead of workspace dependencies for private usage?